apps/classroomio-com/src/routes/bootcamps/+page.svelte

```diff
@@ -83,7 +83,7 @@
         class="w-full flex flex-col lg:items-center items-start lg:justify-center justify-start gap-y-4"
       >
         <h1
-          class="w-[100%] mx-auto m-0 text-4xl lg:text-7xl md:text-6xl font-bold text-slate-900 lg:text-center"
+          class="w-full mx-auto m-0 text-4xl lg:text-7xl md:text-6xl font-bold text-slate-900 lg:text-center"
```

Comment:
remember i mentioned block level elements taking the full width by default? so the w-full is unnecessary here

Reply:
That has already been removed